<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatalogWindow</class>
 <widget class="QWidget" name="CatalogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>683</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>CatalogWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../toolbar.qrc">
    <normaloff>:/icons/catalog_icon</normaloff>:/icons/catalog_icon</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton {
    border-style: none;
    padding 0px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QMenuBar" name="menuBar">
     <widget class="QMenu" name="menu_View">
      <property name="title">
       <string>&amp;View</string>
      </property>
      <addaction name="actionFolderViewList"/>
      <addaction name="actionFolderViewIcon"/>
      <addaction name="actionFolderViewIconNoText"/>
      <addaction name="separator"/>
     </widget>
     <widget class="QMenu" name="menu_File">
      <property name="title">
       <string>&amp;File</string>
      </property>
      <addaction name="actionManageCatalogs"/>
      <addaction name="separator"/>
      <addaction name="actionCloseWindow"/>
     </widget>
     <addaction name="menu_File"/>
     <addaction name="menu_View"/>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QToolButton" name="folderViewButton">
      <property name="maximumSize">
       <size>
        <width>27</width>
        <height>26</height>
       </size>
      </property>
      <property name="toolTip">
       <string>FolderViewSelect</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleName">
       <string>FolderViewSelect</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../toolbar.qrc">
        <normaloff>:/icons/folder_view_icon_blue</normaloff>:/icons/folder_view_icon_blue</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="reloadButton">
      <property name="minimumSize">
       <size>
        <width>27</width>
        <height>26</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>27</width>
        <height>26</height>
       </size>
      </property>
      <property name="toolTip">
       <string>ManageCatalogs</string>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../toolbar.qrc">
        <normaloff>:/icons/catalog_icon</normaloff>:/icons/catalog_icon</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QComboBox" name="searchCombo">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="tagFrame">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: lightskyblue;
    border-style: inset;
    border-width: 1px;
    border-radius: 6px;
/*    border-color: beige; */
    font: 11px;
/*    min-width: 10em; */
    padding: 3px;
}
QPushButton:checked {
    background-color: lightpink;
    border-style: outset;
    border-width: 1px;
    border-radius: 6px;
/*    border-color: beige;*/
    font: 11px;
/*    min-width: 10em; */
    padding: 3px;
}
QFrame.layout {
  layoutLeftMargin: 1px;
  layoutRightMargin: 1px;
  layoutTopMargin: 1px;
  layoutBottomMargin: 1px;
  layoutSpacing: 1px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string notr="true">PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="volumeList">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #C0C0C0</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="iconSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>true</bool>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string notr="true">(StatusLabel)</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStatusBar" name="statusBar"/>
   </item>
  </layout>
  <action name="actionPrev">
   <property name="icon">
    <iconset resource="../toolbar.qrc">
     <normaloff>:/icons/arrow_left_icon</normaloff>:/icons/arrow_left_icon</iconset>
   </property>
   <property name="text">
    <string>Prev</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../toolbar.qrc">
     <normaloff>:/icons/arrow_right_icon</normaloff>:/icons/arrow_right_icon</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionFolderViewList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../toolbar.qrc">
     <normaloff>:/icons/folder_view_list_blue</normaloff>:/icons/folder_view_list_blue</iconset>
   </property>
   <property name="text">
    <string>FolderViewList</string>
   </property>
  </action>
  <action name="actionFolderViewIcon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../toolbar.qrc">
     <normaloff>:/icons/folder_view_icon_red</normaloff>:/icons/folder_view_icon_red</iconset>
   </property>
   <property name="text">
    <string>FolderViewIcon</string>
   </property>
  </action>
  <action name="actionFolderViewIconNoText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../toolbar.qrc">
     <normaloff>:/icons/folder_view_icon_green</normaloff>:/icons/folder_view_icon_green</iconset>
   </property>
   <property name="text">
    <string>FolderViewIconNoText</string>
   </property>
  </action>
  <action name="actionFolderViewSelect">
   <property name="icon">
    <iconset resource="../toolbar.qrc">
     <normaloff>:/icons/folder_view_icon_blue</normaloff>:/icons/folder_view_icon_blue</iconset>
   </property>
   <property name="text">
    <string>FolderViewSelect</string>
   </property>
  </action>
  <action name="actionManageCatalogs">
   <property name="icon">
    <iconset resource="../toolbar.qrc">
     <normaloff>:/icons/catalog_icon</normaloff>:/icons/catalog_icon</iconset>
   </property>
   <property name="text">
    <string>ManageCatalogs</string>
   </property>
  </action>
  <action name="actionCloseWindow">
   <property name="text">
    <string>&amp;CloseWindow</string>
   </property>
  </action>
  <action name="actionTitleWithoutOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove (Options) from Book Title</string>
   </property>
  </action>
  <action name="actionSearchTitleWithOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove (Options) from book title when searching</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../toolbar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>folderViewButton</sender>
   <signal>clicked()</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_folderViewSelect_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchCombo</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_searchTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFolderViewIcon</sender>
   <signal>triggered()</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_folderViewIcon_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFolderViewIconNoText</sender>
   <signal>triggered()</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_folderViewNotext_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFolderViewList</sender>
   <signal>triggered()</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_folderViewList_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reloadButton</sender>
   <signal>clicked()</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_manageDatabase_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_searchTextIndexChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCloseWindow</sender>
   <signal>triggered()</signal>
   <receiver>CatalogWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionManageCatalogs</sender>
   <signal>triggered()</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_manageDatabase_triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CatalogWindow</receiver>
   <slot>on_itemDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_treeItemChanged(QString)</slot>
  <slot>on_folderViewSelect_triggered()</slot>
  <slot>on_searchTextChanged(QString)</slot>
  <slot>on_folderViewList_triggered()</slot>
  <slot>on_folderViewIcon_triggered()</slot>
  <slot>on_folderViewNotext_triggered()</slot>
  <slot>on_manageDatabase_triggered()</slot>
  <slot>on_itemDoubleClicked(QListWidgetItem*)</slot>
  <slot>on_searchTextFinished()</slot>
  <slot>on_searchTextIndexChanged(QString)</slot>
  <slot>on_searchTitleWithOptions_triggered(bool)</slot>
  <slot>on_catalogTitleWithoutOptions_triggered(bool)</slot>
  <slot>on_itemDoubleClicked(QModelIndex)</slot>
 </slots>
</ui>
